driver/pad_constant_driver.hpp

```diff
@@ -0,0 +1,452 @@
+
+    int RunForwardGPU() override;
+    int RunForwardCPU();
+
+    int RunBackwardGPU() override;
+    int RunBackwardCPU();
+
+    Tref GetTolerance();
+    int VerifyBackward() override;
+    int VerifyForward() override;
+    ~ConstantPadDriver() override
+    {
+        miopenDestroyTensorDescriptor(inputDesc);
+        miopenDestroyTensorDescriptor(outputDesc);
+        miopenDestroyTensorDescriptor(inputGradDesc);
+        miopenDestroyTensorDescriptor(outputGradDesc);
+    }
+
+private:
+    InputFlags inflags;
+
+    int forw;
+
+    miopenTensorDescriptor_t inputDesc;
+    miopenTensorDescriptor_t inputGradDesc;
+    miopenTensorDescriptor_t outputDesc;
+    miopenTensorDescriptor_t outputGradDesc;
+
+    std::unique_ptr<GPUMem> input_dev;
+    std::unique_ptr<GPUMem> input_grad_dev;
+    std::unique_ptr<GPUMem> output_dev;
+    std::unique_ptr<GPUMem> output_grad_dev;
+
+    std::vector<Tgpu> input;
+    std::vector<Tgpu> input_grad;
+    std::vector<Tgpu> output;
+    std::vector<Tgpu> output_grad;
+
+    std::vector<Tref> output_host;
+    std::vector<Tref> input_grad_host;
+
+    std::vector<int64_t> padding;
+    Tgpu value;
+
+    bool is_contiguous;
+};
+
+// Equivalent tensor.transpose(0, -1).contiguous().transpose(0, -1)
+template <typename Tgpu, typename Tref>
+std::vector<int> ConstantPadDriver<Tgpu, Tref>::ComputeStrides(std::vector<int> inputDim)
+{
+    if(!is_contiguous)
+        std::swap(inputDim.front(), inputDim.back());
+
+    std::vector<int> strides(inputDim.size());
+    strides.back() = 1;
+    for(int i = inputDim.size() - 2; i >= 0; --i)
+        strides[i] = strides[i + 1] * inputDim[i + 1];
+    if(!is_contiguous)
+        std::swap(strides.front(), strides.back());
+    return strides;
+}
+
+template <typename Tgpu, typename Tref>
+int ConstantPadDriver<Tgpu, Tref>::AddCmdLineArgs()
+{
+    inflags.AddInputFlag("forw",
+                         'F',
+                         "1",
+                         "Run only Forward (1), Run only Backward (2) or Run both Forward and "
+                         "Backward (0) (Default=1)",
+                         "int");
+    inflags.AddTensorFlag("input-dims",
+                          'D',
+                          "1x1x1x8x8",
+                          "The dimensional lengths of the input tensor (Default=1x1x1x8x8)");
+    inflags.AddInputFlag("contiguous", 'C', "0", "Tensor is contiguous or not (Default=0)", "int");
+    inflags.AddInputFlag(
+        "padding-size",
+        'p',
+        "2",
+        "Padding size, where padding_size/2 <= input_size and padding_size is even (Default=2)",
+        "int");
+    inflags.AddInputFlag("padding-value", 'v', "0", "Padding value (Default=0)", "float");
+    inflags.AddInputFlag("iter", 'i', "10", "Number of Iterations (Default=10)", "int");
+    inflags.AddInputFlag("verify", 'V', "1", "Verify (Default=1)", "int");
+    inflags.AddInputFlag("time", 't', "1", "Time (Default=1)", "int");
```

Comment:
change to default is not calculate kernel time
```suggestion
    inflags.AddInputFlag("time", 't', "0", "Time (Default=0)", "int");
```